src/Mod/Part/App/AppPartPy.cpp

```diff
@@ -106,6 +106,8 @@
 
 #ifdef FCUseFreeType
 #  include "FT2FC.h"
+#include "TopoShapeOpCode.h"
+#include "TopoShapeMapper.h"

```

Comment:
Is it really correct that these includes are inside this ifdef? They seem unrelated to FreeType.

Reply:
CLion, I am never again trusting your cute resolve missing definitions button.  I literally never looked up there.  My bad.